src/SwedbankPay.Sdk.Infrastructure/Payments/SwishPayment.cs

```diff
@@ -1,44 +1,45 @@
-﻿using SwedbankPay.Sdk.Extensions;
-using System;
-using System.Net.Http;
-using System.Threading.Tasks;
-
-namespace SwedbankPay.Sdk.Payments.SwishPayments
-{
-    public class SwishPayment
-    {
-        private SwishPayment(SwishPaymentPaymentResponse paymentResponse, HttpClient client)
-        {
-            PaymentResponse = paymentResponse.Payment;
-            var operations = new SwishPaymentOperations(paymentResponse.Operations, client);
-            Operations = operations;
-        }
-
-
-        public SwishPaymentOperations Operations { get; }
-
-        public PaymentResponseObject PaymentResponse { get; }
-
-
-        internal static async Task<SwishPayment> Create(SwishPaymentRequest paymentRequest,
-                                                   HttpClient client,
-                                                   string paymentExpand)
-        {
-            var url = new Uri($"/psp/swish/payments{paymentExpand}", UriKind.Relative);
-
-            var paymentResponse = await client.PostAsJsonAsync<SwishPaymentPaymentResponse>(url, paymentRequest);
-            return new SwishPayment(paymentResponse, client);
-        }
-
-
-        internal static async Task<SwishPayment> Get(Uri id, HttpClient client, string paymentExpand)
-        {
-            var url = !string.IsNullOrWhiteSpace(paymentExpand)
-                ? new Uri(id.OriginalString + paymentExpand, UriKind.RelativeOrAbsolute)
-                : id;
-
-            var paymentResponse = await client.GetAsJsonAsync<SwishPaymentPaymentResponse>(url);
-            return new SwishPayment(paymentResponse, client);
-        }
-    }
+﻿using SwedbankPay.Sdk.Extensions;
+using SwedbankPay.Sdk.Payments.SwishPayments;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SwedbankPay.Sdk.Payments
+{
+    public class SwishPayment : ISwishPayment

```

Comment:
Codacy found an issue: [This class can't be instantiated; make its constructor 'public'.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)